<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TRPL-PUN-0115" failures="0" tests="29" name="criticalTestScripts.ApprovalInCritical" time="907.678" errors="2" timestamp="2024-09-04T13:42:26 IST" skipped="26">
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="Upcoming_PieChartPeriod" time="710.259"/>
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="browserClosing" time="143.359">
    <error message="java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
Build info: version: &#039;3.141.0&#039;, revision: &#039;2ecb7d9a&#039;, time: &#039;2018-10-31T20:09:30&#039;
System info: host: &#039;TRPL-PUN-0115&#039;, ip: &#039;172.29.208.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
Build info: version: '3.141.0', revision: '2ecb7d9a', time: '2018-10-31T20:09:30'
System info: host: 'TRPL-PUN-0115', ip: '172.29.208.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '20'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:452)
at login.BasePage.closeBrowser(BasePage.java:303)
at criticalTestScripts.ApprovalInCritical.browserClosing(ApprovalInCritical.java:3314)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 36 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:673)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:535)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
at java.base/java.net.Socket.connect(Socket.java:666)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 56 more
]]>
    </error>
  </testcase> <!-- browserClosing -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="NotCompleted_PieChart" time="54.059">
    <error message="java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
Build info: version: &#039;3.141.0&#039;, revision: &#039;2ecb7d9a&#039;, time: &#039;2018-10-31T20:09:30&#039;
System info: host: &#039;TRPL-PUN-0115&#039;, ip: &#039;172.29.208.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;20&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
Build info: version: '3.141.0', revision: '2ecb7d9a', time: '2018-10-31T20:09:30'
System info: host: 'TRPL-PUN-0115', ip: '172.29.208.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '20'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElementsByCssSelector(RemoteWebDriver.java:424)
at org.openqa.selenium.By$ByCssSelector.findElements(By.java:441)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:311)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.frame(RemoteWebDriver.java:878)
at org.openqa.selenium.support.ui.ExpectedConditions$16.apply(ExpectedConditions.java:476)
at org.openqa.selenium.support.ui.ExpectedConditions$16.apply(ExpectedConditions.java:472)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
at approval.ApprovalcountPOM.GraphCountIn(ApprovalcountPOM.java:2223)
at criticalTestScripts.ApprovalInCritical.NotCompleted_PieChart(ApprovalInCritical.java:389)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:49124
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 37 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:673)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:535)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:585)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
at java.base/java.net.Socket.connect(Socket.java:666)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 57 more
]]>
    </error>
  </testcase> <!-- NotCompleted_PieChart -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="ClosedTimely_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedTimely_PieChart -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="ClosedDelayed_PieChart" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedDelayed_PieChart -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="NotApplicable_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- NotApplicable_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="Overdue_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- Overdue_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="dueToday_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- dueToday_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="PFR_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- PFR_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="inProgress_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- inProgress_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="Rejected_PieChartInternal" time="0.000">
    <skipped/>
  </testcase> <!-- Rejected_PieChartInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="BargraphBSECriticalStatutory" time="0.001">
    <skipped/>
  </testcase> <!-- BargraphBSECriticalStatutory -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="BargraphBSEHighStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- BargraphBSEHighStatutory -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="BargraphBSEMediumStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- BargraphBSEMediumStatutory -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="BargraphBSELowStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- BargraphBSELowStatutory -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="RiskSummaryCriticalInternal" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryCriticalInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="ClosedTimely_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedTimely_PieChartPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="RiskSummaryHighInternal" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryHighInternal -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="RiskSummaryMediumStatutory" time="0.000">
    <skipped/>
  </testcase> <!-- RiskSummaryMediumStatutory -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="ClosedDelayed_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- ClosedDelayed_PieChartPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="NotCompleted_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- NotCompleted_PieChartPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="NotApplicable_PieChartPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- NotApplicable_PieChartPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="Overdue_PieChartInternalPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- Overdue_PieChartInternalPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="PFR_PieChartInternalPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- PFR_PieChartInternalPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="InProgress_PieChartInternalPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- InProgress_PieChartInternalPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="Rejected_PieChartInternalPeriod" time="0.000">
    <skipped/>
  </testcase> <!-- Rejected_PieChartInternalPeriod -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="complianceCalendar" time="0.000">
    <skipped/>
  </testcase> <!-- complianceCalendar -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="DetailedReportIn" time="0.000">
    <skipped/>
  </testcase> <!-- DetailedReportIn -->
  <system-out/>
  <testcase classname="criticalTestScripts.ApprovalInCritical" name="ComplianceDocuments" time="0.000">
    <skipped/>
  </testcase> <!-- ComplianceDocuments -->
  <system-out/>
</testsuite> <!-- criticalTestScripts.ApprovalInCritical -->
